lib/index.ts

```diff
@@ -174,6 +266,14 @@ export class Front {
 
 		return newPath;
 	}
+
+	private validateEventSignature(data: any, signature: string): boolean {
+		const hash = crypto.createHmac('sha1', this.apiSecret)
+			.update(JSON.stringify(data))
```

Comment:
`JSON.stringify` can throw, which means that any caller needs to handle that (or risk the front-sdk crashing the nodejs process - maybe that's already the case, but if you can check that'd be great